XenAdmin/ConsoleView/RdpClient.cs

```diff
@@ -55,11 +55,8 @@ class RdpClient: IRemoteConsole
         /// </summary>
         private MsRdpClient9 rdpClient9 = null;
 
-        private MsRdpClient6 rdpClient6 = null;
-
         /// <summary>
-        /// This will be equal to rdpClient9, if the DLL that we've got is version 8, otherwise equal to
-        /// rdpClient6.
+        /// This will be equal to rdpClient9

```

Comment:
Hmm, yes, I was thinking about it, but perhaps we may want to leave it there in case there are new guests in future for which a new "rdpClientx" is required?